### PR TITLE
Docs: how to update models

### PR DESCRIPTION
**Pull request by nanjiangshu**
This PR is related to the issue [#970](https://app.zenhub.com/workspaces/metatlas-sprint-607745622d49260019ce115a/issues/metabolicatlas/metabolicatlas/970)

It adds instruction about how to update models manually

### CODE REVIEW

**Review by inghylt — APPROVED** [review]
Nice!

**Review by e0 — APPROVED** [review]
This looks great @nanjiangshu!

It would be nice if there can be a bit of info included about installing and perhaps altering the run command. I believe these problems are macOS specific.

For installing, I had a problem with `pkg-config` when running `pip install -r ./utils/requirements.txt`. After installing with `brew install pkg-config`, I could install the requirements.

For running, I had to run it with `python ./utils/fetch_release_data.py -u`.

File: doc/HOW-TO-UPDATE-MODELS.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  ```
+
+- Select a target model.
+
+- Clone the target model from [SysBioChalmers](https://github.com/SysBioChalmers).
+
+- Check out the latest version.
+
+- Copy the YAML file and all TSV files (inside the subfolder `model`) to the
+   folder `integrated-models/xxx-GEM` in the repo `data-files`, where `xxx-GEM`
```
Comment (mihai-sysbio) [inline doc/HOW-TO-UPDATE-MODELS.md:35]:
```suggestion
- Copy the YAML file and all TSV files (inside the subfolder `model`) to the
   folder `integrated-models/xyz-GEM` in the repo `data-files`, where `xyz-GEM`
```

File: doc/HOW-TO-UPDATE-MODELS.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   sure the `version` and the `date` in the JSON file match the values in the
+   YAML file.
+
+- Run `./generate` in the repo `data-generation`. If there are any error
+   messages, fix the model files according to the message. Repeat until no
+   error message is shown with data generation.
+
+- Re-build and re-deploy the stack locally. Then run the test by 
+   ```ma-exec api yarn test``` and play around with the web pages.
+   If there are any abnormal behaviors found, try to solved it.
```
Comment (MalinAhlberg) [inline doc/HOW-TO-UPDATE-MODELS.md:56]:
```suggestion
   If there are any abnormal behaviors found, try to solve it.
```

**Review by MalinAhlberg — APPROVED** [review]
Very good documentation :+1: . Only found a typo.

File: doc/HOW-TO-UPDATE-MODELS.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  ```
+  Yeast-GEM can be updated: 8.4.2 => 8.6.0
+  Human-GEM can be updated: 1.10.0 => 1.11.0
+  Mouse-GEM can be updated: 1.2.0 => 1.3.0
+  Rat-GEM can be updated: 1.2.0 => 1.3.0
+  Zebrafish-GEM can be updated: 1.1.0 => 1.2.0
+  Fruitfly-GEM can be updated: 1.1.0 => 1.2.0
+  Worm-GEM can be updated: 1.1.0 => 1.3.0
+  ```
```
[the remark this answers is not in the document]
Comment (nanjiangshu) [reply doc/HOW-TO-UPDATE-MODELS.md]: The script is a requirement for the third point of the acceptance criteria 
Issue [#1004](https://app.zenhub.com/workspaces/metatlas-sprint-607745622d49260019ce115a/issues/metabolicatlas/metabolicatlas/1004) is created for integrating this script in the main repo.

**Pull request merged**